tests/UI/pages/commonPage.js

```diff
@@ -22,6 +22,24 @@ class CommonPage {
     await page.goto(url);
   }
 
+  /**
+   * Go to FO page
+   * @param page {Page} Browser tab
+   * @return {Promise<void>}
+   */
+  async goToFo(page) {
+    await this.goTo(page, global.FO.URL);
+  }
+
+  /**
+   * Go to BO page
+   * @param page {Page} Browser tab
+   * @return {Promise<void>}
+   */
+  async goToBO(page) {
+    await this.goTo(page, global.BO.URL);
+  }
+
```

Comment:
Do you really need this? goTo is enough, as you can use the global variable, this abstraction looks overkill

Reply:
This is a good thing because it's the most important thing in a Page object model architecture : Separate page logic from tests logic. The aim is not to call the URL in a test file.